install-dev/controllers/http/process.php

```diff
@@ -91,8 +91,6 @@ public function process()
                 $this->processInstallDefaultData();
             } elseif (Tools::getValue('populateDatabase') && !empty($this->session->process_validated['installDatabase'])) {
                 $this->processPopulateDatabase();
-                // download and install language pack
-                Language::downloadAndInstallLanguagePack($this->session->lang);
```

Comment:
Removed because unreachable. The language is actually installed in the previous step.